pages/Configuring/Configuring-Hyprland.md

```diff
@@ -1,7 +1,7 @@
 The config is located in `~/.config/hypr/hyprland.conf`.
 
 Hyprland will automatically generate an example config for you if
-you don't have one.
+you don't have one. you don't have one. Or you can find an example config [here](https://github.com/hyprwm/Hyprland/blob/main/example/hyprland.conf).
```

Comment:
?

Reply:
```suggestion
you don't have one. Or you can find an example config [here](https://github.com/hyprwm/Hyprland/blob/main/example/hyprland.conf).
```

Good catch, where you talking about that? Or about something else?

Reply:
yes. Also, don't begin a sentence with "or". I'd rather "You can find the example config ..."